infrastructure/src/main/java/org/corfudb/infrastructure/ManagementAgent.java

```diff
@@ -709,8 +729,10 @@ private void correctOutOfPhaseEpochs(PollReport pollReport) {
             // Retrieve the correct layout from quorum of members to reseal servers.
             // If we are unable to reach a consensus from a quorum we get an exception and
             // abort the epoch correction phase.
-            Layout quorumLayout = fetchQuorumLayout(layoutCompletableFutureMap.values()
-                    .toArray(new CompletableFuture[layoutCompletableFutureMap.size()]));
+            CompletableFuture[] layoutFutures = layoutCompletableFutureMap
+                    .values()
+                    .toArray(new CompletableFuture[layoutCompletableFutureMap.size()]);
+            Layout quorumLayout = fetchQuorumLayout(layoutFutures);
```

Comment:
If this is only for better readability then we could also have formatted the code with proper indentation.

Reply:
It is a proper indentation. Formatted by intellij idea